frontend/src/PlayersView/PlayersView.jsx

```diff
@@ -1,28 +1,80 @@
-import StatsTable from "./StatsTable";
+import StatsTable from "../TableComponents/StatsTable";
+import SortMenu from "../TableComponents/SortMenu";
 import Header from "../Header/Header";
+import SearchBar from "../TableComponents/SearchBar";
```

Comment:
- If you didn't want to deal with relative imports (the `../` part) you could add a config to use absolute imports. Totally optional, but that way you can have more nested directories if you ever get there https://medium.com/geekculture/making-life-easier-with-absolute-imports-react-in-javascript-and-typescript-bbdab8a8a3a1
- nit: Typically the naming conventions for directories is snake case with either underscore or hyphen so like `table_components`, `players_view`, `single_player`, etc

Reply:
Fixed the nit. Since I don't plan on adding more components, I will save the absolute imports for after I've finalized my mvp